content/docs/2.8/scalers/prometheus.md

```diff
@@ -23,6 +23,7 @@ triggers:
     # Optional fields:
     namespace: example-namespace  # for namespaced queries, eg. Thanos
     cortexOrgId: my-org # Optional. X-Scope-OrgID header for Cortex.
+    ignoreNullValues: disable # Default is enable, which means ignoring the empty value list from prometheus. Set to disable the scaler will return error when prometheus target is lost 
```

Comment:
As per the code changes, let's use `true`/`false` instead

Reply:
update to true/false already